cmd/validate_command_flags_test.go

```diff
@@ -0,0 +1,50 @@
+package cmd
```

Comment:
I think this test would be better suited to the acceptance suite rather than expanding the tests that exist in the `cmd` pacakge. I'm not quite sure what the representative set of commands and their permutations are, but I'd like us to test a handful of the actual commands rather than recreating and unit testing them

Reply:
I think the loadConfigFile_test was a bit of a misdirect on what convention to follow here.